### ISSUE TITLE
Combatant Health

### ISSUE THREAD

**Issue by NathanBroadway** [Dreadpirate71/DungeonApp#1]
Does it make sense that combatants can have negative health?
[https://github.com/Dreadpirate71/DungeonApp/blob/main/Dungeon%20App/Combatant.cs#L41](Combatant)

**Comment by Dreadpirate71** [comment]
not really. I play world of warcraft and in that the you can do more damage than needed to kill the monster but the health is never dispayed as negative.  I need to change this so that it just goes to zero. Thanks Nathan for looking at my code. I really appreciate your input :)
